icepyx/quest/dataset_scripts/argo.py

```diff
@@ -56,9 +56,9 @@ def __str__(self):
             df = "\n" + str(self.argodata.head())
         s = (
             "---Argo---\n"
-            "Parameters: {0}\n"
-            "Pressure range: {1}\n"
-            "Dataframe head: {2}".format(self.params, prange, df)
+            f"Parameters: {self.params}\n"
+            f"Pressure range: {prange}\n"
+            f"Dataframe head: {df}"
```

Comment:
So much more readable. The implicit concatenation combined with `.format()` just weirds me out :)